processing/src/main/java/org/apache/druid/query/metadata/metadata/ColumnAnalysis.java

```diff
@@ -73,6 +78,12 @@ public String getType()
     return type;
   }
 
+  @JsonProperty
+  public ColumnType getTypeSignature()
```

Comment:
Consider placing this getter above `getType`, which I think will put it earlier in the JSON and make it seem more "official".